source/utilities.cc

```diff
@@ -2468,6 +2468,26 @@ namespace aspect
 
 
 
+    Operator::Operator(const std::string &operation)
+    {
+      // create operator list
+      if (operation == "add")
+        op = Operator::add;
+      else if (operation == "subtract")
+        op = Operator::subtract;
+      else if (operation == "minimum")
+        op = Operator::minimum;
+      else if (operation == "maximum")
+        op = Operator::maximum;
+      else
+        AssertThrow(false,
+                    ExcMessage ("ASPECT only accepts the following operators: "
+                                "add, subtract, minimum and maximum. But your parameter file "
+                                "contains: " + operation + ". Please check your parameter file.") );
```

Comment:
just write this as
```
  "contains <" + operation + ">. Please check..."
```
for consistency with other places.

In some places, we pass the name of the parameter where this happened to similar functions (e..g., to `possibly_extend_1_to_N()`) so that it's easier to find the location where this happened in the input file. Could we do the same here?